README.md

```diff
@@ -1,67 +1,66 @@
+[Packaged MSIX](https://docs.microsoft.com/windows/msix/overview) desktop apps</br>(WPF, Win32, WinForms, console) |                ✅ [WinUI 3](https://github.com/microsoft/microsoft-ui-xaml/blob/master/docs/roadmap.md#winui-3)</br>✅ [Text rendering](https://docs.microsoft.com/en-us/windows/apps/project-reunion/dwritecore)</br>✅ [Localization](https://docs.microsoft.com/windows/apps/project-reunion/mrtcore/mrtcore-overview)  | ✅ [WinUI 3](https://github.com/microsoft/microsoft-ui-xaml/blob/master/docs/roadmap.md#winui-3)</br>✅ [Text rendering](https://docs.microsoft.com/en-us/windows/apps/project-reunion/dwritecore)</br>✅ [Localization](https://docs.microsoft.com/windows/apps/project-reunion/mrtcore/mrtcore-overview)</br>🔄 [App lifecycle](https://github.com/microsoft/ProjectReunion/issues/111)  | ✅ [WinUI 3](https://github.com/microsoft/microsoft-ui-xaml/blob/master/docs/roadmap.md#winui-3)</br>✅ [Text rendering](https://github.com/microsoft/ProjectReunion/issues/112)</br>✅ [Localization](https://docs.microsoft.com/windows/apps/project-reunion/mrtcore/mrtcore-overview)</br>✅ [App lifecycle](https://github.com/microsoft/ProjectReunion/issues/111)  | 
+Unpackaged desktop apps</br>(WPF, Win32, WinForms, console) | ❌ Not supported  |                    🔄 [Text rendering](https://docs.microsoft.com/en-us/windows/apps/project-reunion/dwritecore)</br>🔄 [Localization](https://docs.microsoft.com/windows/apps/project-reunion/mrtcore/mrtcore-overview)</br>🔄 [App lifecycle](https://github.com/microsoft/ProjectReunion/issues/111)  |  ✅ [WinUI 3](https://github.com/microsoft/microsoft-ui-xaml/blob/master/docs/roadmap.md#winui-3)</br>✅ [Text rendering](https://docs.microsoft.com/en-us/windows/apps/project-reunion/dwritecore)</br>✅ [Localization](https://docs.microsoft.com/windows/apps/project-reunion/mrtcore/mrtcore-overview)</br>✅ [App lifecycle](https://github.com/microsoft/ProjectReunion/issues/111) | 
 
-**Compatible**
+✅ Supported | 🔄 Preview
 
-Project Reunion works in all your apps - Win32, Packaged, and UWP - and across many versions of Windows.
 
-**Modern**
+See our [full roadmap](docs/roadmap.md) for a more detailed roadmap.
 
-Project Reunion supports your app's use of modern software libraries for UI, AI, ML, packaging, frameworks, 
-and libraries. Language projections for C++, Rust, C#, and JavaScript expand the benefits to all your 
-apps.  Cloud-backed apps, streaming, and edge-compute apps are all great places to use Project Reunion's 
-capabilities.
 
-**Agile**
 
-Project Reunion ships out of band with OS releases, with regular previews. You get to incrementally 
-adopt Project Reunion components for your existing apps and middleware libraries using NuGet.
+### Version Support
+
+Project Reunion aims to work on supported versions of Windows. Right now we support down to build 17763 (version 1809/October 2018 Update) of Windows 10. We'll use techniques like '_polyfills_' 

```

Comment:
Please word-wrap at 100 characters. (Tables are the exception given how they work in Markdown)

Reply:
Eh, we have software with word wrapping built in, manually wrapping just creates pain when then later trying to edit. I understand GitHub isn't great at allowing you to inline comment, but that's more of a problem with GitHub itself.

Reply:
This is part of our guidelines - https://github.com/microsoft/ProjectReunion/blob/main/docs/Coding-Guidelines.md#markdown

Use a tool like Prettifier or another markdown formatting / linter that lets you wrap the content, please.  I think VSCode knows how to do this automatically but you might need to configure it.  The docs.microsoft.com authoring pack for VSCode has a bunch of great stuff in it.

Reply:
I don't agree with the suggestion either, as it creates unnecessary formatting tax on edits, but it has already been codified so ... gotta roll with it.

I want to echo that this messaging should be considered _critical_ so to speak. It sends a promise that is in conflict with current messaging. (See my earlier comment about conversations with Rich on this.) Please take this part seriously!